examples/website-demos/social-messaging/src/components/MessageInput/SocialMessageInput.tsx

```diff
@@ -115,27 +115,29 @@ export const SocialMessageInput = (props: Props) => {
             giphyState
           }
         >
-          <EmojiPicker />
           <div className={`input-ui-input ${giphyState ? 'giphy' : ''}`}>
             {giphyState && !numberOfUploads && <GiphyIcon />}
             <UploadsPreview />
-            <ChatAutoComplete onChange={onChange} placeholder='Send a message' />
-            {
-              <>
-                {!giphyState && (
-                  <div
-                    className='input-ui-input-emoji-picker'
-                    ref={emojiPickerRef}
-                    onClick={openEmojiPicker}
-                  >
-                    <EmojiPickerIcon />
-                  </div>
-                )}
-              </>
-            }
+            <div className='input-ui-icons-emoji'>
```

Comment:
this can also be changed, if we're building on the previous classname something like `input-ui-input-textarea` could work, do whatever you think makes the most sense